docs/contributor-guide/pull-request.md

```diff
@@ -1,16 +1,16 @@
-# ↪️ Pull Requests
+# Contributing pull requests
```

Comment:
Somewhere in this section, I think it would be useful to emphasize that the dev tasks should be run before making a PR (or maybe even before each commit) to keep the commit history more readable, and avoid too many commits that fail on the CI jobs.